plugin-openshift/che-plugin-openshift-ext-client/src/main/resources/org/eclipse/che/ide/ext/openshift/client/OpenshiftLocalizationConstant.properties

```diff
@@ -103,6 +103,7 @@ invalid.openshift.project.name.error=Name must be a valid DNS label
 invalid.che.project.name.error=Name must be a valid Che project name
 existing.project.name.error=A project with this name already exists
 
+retrieving.projects.data=Retrieving project''s data...
```

Comment:
Yes, we need it  for handling symbol " ' ". In java we handle spacial symbols with help "  \ "   
-> "some text \\". In properties we use " ' " instead of " \ "